server/lib/config/pbench-server.cfg.example

```diff
@@ -1,6 +1,7 @@
 [DEFAULT]
 # The values here override those in pbench-server-default.cfg.
-# An installation will want to override at least the following:
+# An installation must override at least the "example.com"
+# host names.
 default-host = pbench.example.com
 install-dir = /opt/pbench-server
```

Comment:
Should this entire section or lines 5 and/or 6 be commented out...with the expectation that anyone who wants to override `pbench-server-default.cfg` will uncomment them and change their values to something other than the example placeholders?

Possibly ditto the other `host` and/or `port` lines, below?

Reply:
This file strides between two different styles but doesn't really conform to either.

1. We could build it as a documented template of all of the config options you _could_ set or change, with values commented out. I think Apache does this with httpd.
2. We could keep it stripped down and clean with only the config options a site really _has_ to specify, like `default-host` ... in which case commenting them just means they need to remove the `#` when they edit it and it doesn't have much value.

We kinda do both here. For example, `install-dir` had better be in `pbench-server-default.cfg` because that's the standard default. They'd change it only if they do something really weird; and either it shouldn't be here at all or it should be commented out. The `default-host` is something they have to set.

There are no default, for example, for the `[Indexing]` section below, nor any guidelines for selecting the `bulk_action_count` (or even what it means). Arguably, that _should_ be a default, and should be documented.

I hope you won't choose to drag me down this rat hole right now, but it's something we really ought to record and think about as part of turning our project into a product.

Reply:
No dragging.  (See above where I approved the PR!)

However, it does sound like you would like to see an open an issue for this....  ;-)

Reply:
Yes, and thanks.